pkg/naming/main.go

```diff
@@ -15,18 +15,14 @@
 // Package naming is for determining the names for components (containers, services, ...).
 package naming
 
-import (
-	"github.com/open-telemetry/opentelemetry-operator/apis/v1alpha1"
-)
-
 // ConfigMap builds the name for the config map used in the OpenTelemetryCollector containers.
-func ConfigMap(otelcol v1alpha1.OpenTelemetryCollector) string {
-	return DNSName(Truncate("%s-collector", 63, otelcol.Name))
+func ConfigMap(otelcol string) string {
```

Comment:
Changing this to string to avoid circular imports.

Reply:
This is great :)